docs/modules/ROOT/pages/production-checklist.adoc

```diff
@@ -70,7 +70,7 @@ General recommendations:
 * GC logs should be enabled
 * Minimum and maximum heap size should be equal
 
-For Java 11+:
+For Java 17+:
```

Comment:
Nit - not sure what value this adds? There's no non-Java 17 option for the user, or in the documentation